### PR TITLE
change: users/demo file content change

### PR DESCRIPTION
**Pull request by yasin-yilmaz**
better content for demo

### CODE REVIEW

File: users/demo.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-Dolor mollit ea ad nisi minim.
\ No newline at end of file
+Dolor mollit ea ad nisi minim.
+
+Consequat mollit nulla irure laborum reprehenderit deserunt laborum laborum esse ullamco.
```
Comment (fagah6) [inline users/demo.txt]: this line too long split it two line

Comment (fagah6) [reply users/demo.txt]: demo

**Review by fagah6 — CHANGES_REQUESTED** [review]
i add some things please check

**Comment by yasin-yilmaz** [comment]
done

**Pull request merged**